pipelines/concourse-pipelines.yml

```diff
@@ -61,6 +61,7 @@ jobs:
             gcp-project-name: census-rm-blacklodge
             kubernetes-cluster-name: rm-k8s-cluster
             acceptance-tests-image: eu.gcr.io/census-gcr-rm/rm/census-rm-acceptance-tests:latest
+            release-directory: releases/version-0
```

Comment:
might be a bit nicer if we just specify a version here, the directory path can be implied from that??

Reply:
That was the original design but I found if it's set this way (arguably a little longer) then it could also be omitted to test against root configs.

Reply:
ah yeah thats a good point 👍